tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -6,6 +6,14 @@
     ceph_backend_configuration_fsid_shell_vars: |
       CEPH_FSID=$(oc get secret ceph-conf-files -o json | jq -r '.data."ceph.conf"' | base64 -d | grep fsid | sed -e 's/fsid = //')
 
+- name: Restart neutron pods after control plane adoption
```

Comment:
As a workaround LGTM but can we create a Jira to track the root cause and link it from here? As it is now i think this would fix CI but not the docs, we could forget about this problem and later we'd get the same bug reported by an end user...

If we want to fix CI quickly we can add the link in a follow up PR and merge this one.

Reply:
@jistr root cause is basically how we adapt and create endpoints, reported https://issues.redhat.com/browse/OSPRH-10230 explaining this and to track fix on neutron side. Will clear the workaround after fixing OSPRH-10230. So this can be merged to clear adoption jobs for now.